lib/writer.js

```diff
@@ -0,0 +1,89 @@
+const { Feed } = require('@synonymdev/feeds')
+
+const MINUTE = 1000 * 60
+
+const INTERVAL = MINUTE // every 1 minute
+
+class BitcoinFeed extends Feed {
+  /**
+   * @param {ConstructorParameters<typeof Feed>[0]} client
+   * @param {ConstructorParameters<typeof Feed>[1]} config
+   * @param {ConstructorParameters<typeof Feed>[2]} opts
+   */
+  constructor (client, config, opts) {
+    super(client, config, opts)
+
+    this._config = config
+
+    // Time since start
+    this._age = -1
+
+    // Start on next tick, allowing mocking some methods in unit test after construction.
+    setTimeout(() => {
```

Comment:
Does it make sense to move this to Feeds class?

Reply:
This is not really a feature, it helps me mock some methods before the immediate call for `this._onInterval()`. In general implicit logic for intervals is tricky and can throw people off, and hard to generalize for all cases. So let's keep it manual.